src/structures/Role.js

```diff
@@ -82,6 +82,26 @@ class Role extends Base {
      * @type {boolean}
      */
     this.deleted = false;
+
+    /**
+     * The tags this role has
+     * @type {?Object}
+     * @property {?Snowflake} botID The id of the bot this role belongs to
+     * @property {?Snowflake} integrationID The id of the integration this role belongs to
+     * @property {?true} premiumSubscriberRole Whether this is the guild's premium subscription role
```

Comment:
```suggestion
     * @property {true} [premiumSubscriberRole] Whether this is the guild's premium subscription role
```
i think this is technically correct here, as the `?` indicates it can be null, at the least it should be marked optional with `[]` though

same for the 2 lines above

Reply:
according to vlad this needs both ? as well as []

Reply:
Why would it need both? It's not nullable, just optional

Reply:
Oh, true, welcome to 
![image](https://user-images.githubusercontent.com/17960496/100288975-88a89100-2f80-11eb-83c2-563bded1fbf0.png)

Reply:
same, resolved